src/utils/coroutine.js

```diff
@@ -103,6 +103,11 @@ export function cancelable(iter, signal) {
   })();
 }
 
+const nextFramePromise = Promise.resolve();
+export function nextFrame() {
+  return nextFramePromise;
+}
```

Comment:
The name `nextFrame` is a little weird to me, since it assumes information about the calling context (i.e. this coroutine that ticks every frame). It's accurate everywhere we've wanted to use it, but still weird to me. I don't have a better name in mind though.

Reply:
Yeah. This is part of the "courtine API" like timers stuff we experimented with so I guess I will add `cr`. I don't like that much either but at least its consistent. Maybe we want to do something like `Coroutine.nextFrame()`